app/mailers/app_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class AppMailer < ActionMailer::Base
+  def notify_user_create(user)
+    @user = user
+    mail to: @user.email, from: 'noreply@myflix2622.herokuapp.com', subject: "Welcome to Myflix"
+  end
+
+  def notify_password_reset(user)
+    @user = user
+    mail to: @user.email, from: 'noreply@myflix2622.herokuapp.com', subject: "Myflix: Password Reset"
```

Comment:
You can set things like `from` in the defaults, then you don't have to keep repeating yourself.